Public/Src/Engine/Scheduler/Scheduler.cs

```diff
@@ -344,6 +344,26 @@ private void StartWorkers(LoggingContext loggingContext)
             m_allWorker = new AllWorker(m_workers.ToArray());
 
             ExecutionLog?.WorkerList(new WorkerListEventData { Workers = m_workers.SelectArray(w => w.Name) });
+        }
+
+        private readonly TaskSourceSlim<bool> m_schedulerCompletionExceptMaterializeOutputs = TaskSourceSlim.Create<bool>();

```

Comment:
>ate [](start = 12, length = 3)

nit: why do we have a mix of fields and methods in this class? shouldn't we keep all the fields at the top followed by the methods?

Reply:
no particular reason. as Scheduler class gets bigger, people wanted to put the new private members close to where they used it. 

---
In reply to: [348743741](https://github.com/microsoft/BuildXL/pull/1216#discussion_r348743741) [](ancestors = 348743741)